modules/virt-creating-pvcs-to-store-default-os-images.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-default-os-images.adoc
+
+[id="virt-creating-pvcs-to-store-default-os-images_{context}"]
+= Creating persistent volume claims to store default OS images
+
+Follow these steps to create a persistent volume claim (PVC), which you use to upload and save default operating system (OS) images.
+
+.Prerequisites
+
+* You must be logged in as a user with the `os-images.kubevirt.io:edit` RBAC role or as an administrator.
+
+.Procedure
+
+. Select a bootable disk that you want to upload and save as a default OS image.
+
+. In the {product-title} web console, click *Storage > Persistent Volume Claims* in the sidebar menu. The *Persistent Volume Claim* page displays.
+
+. Click the *Create Persistent Volume Claim* button and select the *With Data upload form* option.
+
+. Complete the *Upload Data to Persistent Volume Claim* form to create a PVC that is used to upload and save default OS images.
+.. Click *Browse* and locate a bootable image to upload and save as a default OS image.
+.. Select the *Attach this disk to a VirtualMachine operating system* check box.
+.. Select the OS of the bootable disk that you want to upload from the *Operating System* list.
+.. Select the storage class you want to use in the *Storage Class* list.
+.. Enter the size of the PVC you are creating in the *Size* fields.
+.. Select the *Access Mode*.
+.. Click *Create* to create the PVC.
+
++
+The *Persistent Volume Claim Details* screen displays information about the PVC that you created.
+
+.Additional Resources
+* xref:../../../virt/virtual_machines/virtual_disks/virt-uploading-local-disk-images-web.adoc#virt-uploading-local-disk-images-web[Uploading local disk images]
```

Comment:
@bgaydosrh can you remove the xref introduced by this PR? Our guidance on this hasn't changed and we don't allow xrefs in modules.

Reply:
Hi Vikram,

Sure, I can move the Additional Resources section to the assembly and submit the PR today if that works for you. 

This change only needs to be picked to enterprise-4.6 as the "default os images" feature was renamed and replaced for 4.7/CNV 2.6. This older assembly and the 2 attached modules should be gone for 4.7 --- they are not appearing in my updated `master` at the moment, so I think we're good there.

The replacement for this functionality should be merged today in https://github.com/openshift/openshift-docs/pull/28763 once it has another peer review check and final QE approval.

Thanks for noticing this,
Bob

cc: @ctomasko